obspy/core/utcdatetime.py

```diff
@@ -233,7 +233,7 @@ def __init__(self, *args, **kwargs):
                 try:
                     self._ns = value._ns
                 except AttributeError:
-                    self._from_datetime(datetime.datetime.fromtimestamp(
+                    self._from_datetime(datetime.datetime.utcfromtimestamp(
```

Comment:
Explains a bit the erratic passing/failing of tests (on local computer vs. CI) as the previous code depended on local computer time setting (UTC vs. some local time zone).. nice catch @barsch!